### PR TITLE
etcdctlv3: updated readme for make-mirror: modify/remove prefix in dest cluster

### PR DESCRIPTION
**Pull request by sinsharat**
Added the two flags:
1. dest-prefix
2. no-dest-prefix
which were added for feature make-mirror: modify/remove prefix in dest cluster

### CODE REVIEW

File: etcdctl/README.md [hunk 1]
```diff
@@ -671,6 +671,10 @@ the database, the etcd member releases this free space back to the file system.
 
 - prefix -- The key-value prefix to mirror
 
+- dest-prefix --The destination prefix to mirror a prefix to a different prefix in the destination cluster
```
Comment (gyuho) [inline etcdctl/README.md]: Add a whitespace `-- The`?

File: etcdctl/README.md [hunk 2]
```diff
@@ -671,6 +671,10 @@ the database, the etcd member releases this free space back to the file system.
...
+
+- no-dest-prefix --Mirror key-values to the root of the destination cluster
```
Comment (gyuho) [inline etcdctl/README.md]: `-- Mirror`

File: etcdctl/README.md (hunk 1 above)
Comment (sinsharat) [inline etcdctl/README.md]: sure will do the change

File: etcdctl/README.md (hunk 2 above)
Comment (sinsharat) [inline etcdctl/README.md]: sure will do the change

**Comment by sinsharat** [comment]
@gyuho Have done the changes as per your suggestion.
Thanks!

**Comment by sinsharat** [comment]
@gyuho I was thinking of adding the test case as well for the new feature added. What do you suggest?
Shall i add cases in both e2e cases as well as integration or adding to one is sufficient?

**Comment by gyuho** [comment]
@sinsharat I think e2e test would be enough for these new flags. Defer to @xiang90 @heyitsanthony

**Comment by sinsharat** [comment]
@gyuho thanks for the suggestion. For now will add cases only for e2e. If @xiang90 @heyitsanthony suggest to add for integration then will add that as well.
Thank you!

**Pull request merged**